tools/operator/launcher/k8s/connprofile.go

```diff
@@ -108,11 +111,11 @@ func (k K8s) GetK8sExternalIP(config networkspec.Config, serviceName string) (st
 }
 
 //GetK8sServicePort -- To get the port number of a fabric k8s component
-func (k K8s) GetK8sServicePort(serviceName, serviceType string, forHealth bool) (string, error) {
+func (k K8s) GetK8sServicePort(serviceName, serviceType, namespace string, forHealth bool) (string, error) {
 
 	var k8Ports K8Ports
 	var portNumber int
-	args := []string{"get", "-o", "json", "services", serviceName}
+	args := []string{"-n", namespace, "get", "-o", "json", "services", serviceName}
```

Comment:
Replace this with a call to the SDK (create a separate function for getting this info), we shouldn't be calling the CLI at all anymore.

Reply:
yes replaced with Kubernetes sdk call